solar_car_process.py

```diff
@@ -15,4 +22,25 @@ def start(self):
     
     def check_status(self):
         return self.process.poll()
+
+    
+class Domovoi:
```

Comment:
This class should probably be in a differnet file. I think Putting it in `domovoi.py` makes sense, then we would run it from that file too, kind of like how java does things.